app/models/endpoint.rb

```diff
@@ -22,4 +28,10 @@ def resolve_verify_ssl_value(val)
     else            val
     end
   end
+
+  def self.metrics_default_database_name
+    require 'ovirt_metrics'
+    OvirtMetrics::DEFAULT_HISTORY_DATABASE_NAME
+  end
+  private_class_method :metrics_default_database_name
```

Comment:
This does not belong in the base endpoint.rb.  It should live somewhere in the provider namespace for rhevm.